project_euler/Problem_27/sol1.py

```diff
@@ -11,38 +11,52 @@
 
 import math
 
+
 def isPrime(k):
-	# checks if a number is prime
-	if k < 2: return False
-	elif k == 2: return True
-	elif k % 2 == 0: return False
-	else:
-		for x in range(3, int(math.sqrt(k)+1), 2):
-			if k % x == 0: return False
+    # checks if a number is prime
+    if k < 2:
+        return False
+    elif k == 2:
+        return True
+    elif k % 2 == 0:
+        return False
+    else:
+        for x in range(3, int(math.sqrt(k) + 1), 2):
+            if k % x == 0:
+                return False
 
-	return True
+    return True
 
 
 def solution(a_limit, b_limit):
+    """
+	>>> solution(1000, 1000)
+	-59231
+	>>> solution(2000, 2000)
+	-126479
+	>>> solution(-1000, 1000)
+	0
+	>>> solution(-1000, -1000)
+	0
+
+	"""
```

Comment:
Dedent lines 33 through 42 so they are all left aligned with line 32.
Delete the blank line 41.